public/js/maho-captcha.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Maho
+ *
+ * @package     Maho_Captcha
+ * @copyright   Copyright (c) 2025 Maho (https://mahocommerce.com)
+ * @license     https://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ */
+
+const MahoRecaptcha = {
```

Comment:
hiihihh what a set of changes :-)))

the only thing is that I wouldn't use the name recaptcha, it could be protected but also a bit misleading, i'd use MahoCaptcha probably, what do you think?

Reply:
Oops, you're right, just habit from using Google. 😄

Reply:
renamed it :-)